bika/lims/locales/es/LC_MESSAGES/bika.po

```diff
@@ -5521,15 +5551,15 @@ msgstr ""
 
 #: bika/lims/browser/dashboard/dashboard.py:634
 msgid "Samples to be preserved"
-msgstr ""
+msgstr "Muestras a conservar"
 
 #: bika/lims/browser/dashboard/dashboard.py:650
 msgid "Samples to be received"
-msgstr ""
+msgstr "Muestras a recepcionar"
```

Comment:
@xispa same as before.


 _recibir_ sounds more natural to me, but maybe the technical word is _recepcionar_ so I am ok with whatever you prefer.